infrastructure/web/routes/admin.js

```diff
@@ -1,12 +1,14 @@
 var express = require('express')
 var router = express.Router()
 
+const { getAdminOrders } = require('../controllers/index')
+
 router.get('/', function (req, res) {
   res.render('admin/index')
 })
 
 router.get('/orders', function (req, res) {
-  res.render('admin/orders/index')
+  getAdminOrders(req, res)
```

Comment:
no need for a wrapper function here, you can pass getAdminOrders directly as a handler. But looks like this is an older version of this file. because I already did that and cleaned up.
Make sure always get the latest main branch before starting to add new code https://stackoverflow.com/questions/7244321/how-do-i-update-or-sync-a-forked-repository-on-github